data/README.md

```diff
@@ -1,3 +1,55 @@
 # Test data
+
 This directory contains test data that is uploaded to Azure tests servers in CI. What this means is that when adding
-files in this directory, the `test/sql/test_data_integrity.test` should be updated, otherwise CI will fail.
\ No newline at end of file
+files in this directory, the `test/sql/test_data_integrity.test` should be updated, otherwise CI will fail (see [gen_check_file script](../scripts/gen_check_file.sh)).
+
+## Partitioned
+
+Partitionned data has been add to test the integration of DFS storage account.
+
+Data has been generated from the `l.csv` source file with the following code:
+
+```sh
+spark-shell --master "local[1]"
+```
+
+```scala
+val hadoopConf = spark.sparkContext.hadoopConfiguration
+val fs = org.apache.hadoop.fs.FileSystem.get(hadoopConf)
+fs.setWriteChecksum(false)
+
+spark.conf.set("mapreduce.fileoutputcommitter.marksuccessfuljobs", "false")
+
+spark.read
```

Comment:
I don't think we need this script in the readme.

If you insist, can we please use duckdb instead of spark? ;)

Reply:
My idea was here to use to different system and show that we do have the same result. But i see you point here and I can only agree :)